unpickle.py

```diff
@@ -0,0 +1,12 @@
+import pickle
+import osmnx as ox
+import networkx as nx
+filename = "output/geocode_tree.pkl"
+
+with open(filename, "rb") as f:
+    gt_loaded = pickle.load(f)
+
+print(gt_loaded.nodes())
+shortest_route = nx.shortest_path(
+    gt_loaded, 't-1', 'volley_ball_court', weight='length')
+print(shortest_route)
```

Comment:
load pkl file which we saved using geocodeTree class method
just an example to show later retrieval of the saved data.